app/helpers/admin/orders_helper.rb

```diff
@@ -11,9 +11,26 @@ module OrdersHelper
     def order_adjustments_for_display(order)
       order.adjustments +
         voucher_included_tax_representations(order) +
+        additional_tax_total_representation(order) +
         order.all_adjustments.payment_fee.eligible
     end
 
+    def additional_tax_total_representation(order)
+      adjustment = Spree::Adjustment.where(order_id: order.id, included: false,
+                                           originator_type: 'Spree::TaxRate',
+                                           adjustable_type: 'Spree::Adjustment')
+        .sum(:amount)
+
+      return [] unless adjustment != 0
+
+      [
+        AdjustmentData.new(
+          I18n.t("tax_category"),
```

Comment:
We should use a different label, maybe something like "Tax on fees".
Translations are managed in `config/locales', the new label can be added here : 
https://github.com/openfoodfoundation/openfoodnetwork/blob/15d4f9565d730d038119c5de1ae9375c7b0aef49/config/locales/en.yml#L1061-L1063

Reply:
Unsure about the key name ive used.
i believe the rest of the locales will be addressed by a later Transifex commit right?

Reply:
> i believe the rest of the locales will be addressed by a later Transifex commit right?

`config/locales/en.yml` holds translatable text with the "default" value and we managed translation via Transifex. As part of the release process we update the various locales by pulling data from Transifex.